qtsass/qtsass.py

```diff
@@ -106,31 +202,53 @@ def _recompile(self):
 
     def on_modified(self, event):
         # On Mac, event will always be a directory.
-        # On Windows, only recompile if event's file has the same extension as the input file
-        if event.is_directory and os.path.samefile(event.src_path, self._watched_dir) or (
-           os.path.splitext(event.src_path)[1] == self._watched_extension):
+        # On Windows, only recompile if event's file has
+        #             the same extension as the input file
+        we_should_recompile = (
+            event.is_directory and
+            os.path.samefile(event.src_path, self._watched_dir) or
+            os.path.splitext(event.src_path)[1] == self._watched_extension
+        )
+        if we_should_recompile:
             self._recompile()
 
     def on_created(self, event):
         if os.path.splitext(event.src_path)[1] == self._watched_extension:
             self._recompile()
 
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser(prog="QtSASS",
-                                     description="Compile a Qt compliant CSS file from a SASS stylesheet.",
-                                     )
+def main():
+    parser = argparse.ArgumentParser(
+        prog="QtSASS",
+        description="Compile a Qt compliant CSS file from a SASS stylesheet.",
+    )
     parser.add_argument('input', type=str, help="The SASS stylesheet file.")
-    parser.add_argument('-o', '--output', type=str, help="The path of the generated Qt compliant CSS file.")
-    parser.add_argument('-w', '--watch', action='store_true', help="Whether to watch source file "
-                                                                   "and automatically recompile on file change.")
+    parser.add_argument(
+        '-o',
+        '--output',
+        type=str,
+        help="The path of the generated Qt compliant CSS file."
+    )
+    parser.add_argument(
+        '-w',
+        '--watch',
+        action='store_true',
+        help=(
+            "Whether to watch source file "
```

Comment:
IMO, I find it better to be explicit with boolean flags and the like, to avoid any ambiguity, especially given our highly international audience. I.e. ``"If set, will watch the source file and automatically recompile if it changes."``